app/_src/mesh/guides/deploy-kong-mesh-using-terraform-and-konnect.md

```diff
@@ -0,0 +1,606 @@
+        # (4 unchanged attributes hidden)
+    }
+
+  # konnect_mesh_access_log.konnect_mesh_traffic_permission must be replaced
+-/+ resource "konnect_mesh_traffic_permission" "allow_all" {
+      ~ creation_time     = "2025-03-13T09:53:00.606442Z" -> (known after apply)
+      ~ labels            = {
+          ~ "kuma.io/mesh" = "mesh1" -> "another-name"
+        }
+      ~ mesh              = "mesh1" -> "another-name" # forces replacement
+```
+
+## Deploying a Kubernetes Zone
+
+To deploy a Kubernetes zone you can use any Kubernetes services, in this guide we will use `k3d` to create a local Kubernetes cluster.
+
+Create a new k3d cluster:
+```bash
+k3d cluster create tfmink
+```
+
+Write `tfmink` cluster configuration:
+```bash
+export KUBECONFIG=$(k3d kubeconfig write tfmink)
+```
+
+Update the `main.tf` file to include the time, kubernetes and helm provider:
+
+```hcl
+terraform {
+  required_providers {
+    konnect = {
+      source = "kong/konnect"
+    }
+    konnect-beta = {
+      source  = "kong/konnect-beta"
+      version = "{{ TF_BETA_VERSION }}"
+    }
+    time = {
+      source  = "hashicorp/time"
+      version = "{{ TF_TIME_VERSION }}"
+    }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "{{ TF_KUBERNETES_VERSION }}"
+    }
+    helm = {
+      source  = "hashicorp/helm"
+      version = "{{ TF_HELM_VERSION }}"
+    }
+  }
+}
+```
+
+Add a variable pointing to the Kubeconfig file: 
+
+```hcl
+variable "k8s_cluster_config_path" {
+  type        = string
+  description = "The location where this cluster's kubeconfig will be saved to."
+  default     = "~/.config/k3d/kubeconfig-tfmink.yaml"
+}
+```
+
+You can override that variable by setting the environment variable `TF_VAR_k8s_cluster_config_path`.
+
+We need to create a system account and a token to authenticate the zone:
+
+```hcl
+resource "konnect_system_account" "zone_system_account" {
+  depends_on = [konnect_mesh_control_plane.my_meshcontrolplane]
+  name            = "mesh_${konnect_mesh_control_plane.my_meshcontrolplane.id}_${var.zone_name}"
+  description     = "Terraform generated system account for authentication zone ${var.zone_name} in ${konnect_mesh_control_plane.my_meshcontrolplane.id} control plane."
+  konnect_managed = false
+}
+
+resource "konnect_system_account_role" "zone_system_account_role" {
+  depends_on = [konnect_system_account.zone_system_account]
+  account_id       = konnect_system_account.zone_system_account.id
+  entity_id        = konnect_mesh_control_plane.my_meshcontrolplane.id
+  entity_region    = var.region
+  entity_type_name = "Mesh Control Planes"
+  role_name        = "Connector"
+}
+
+resource "time_offset" "one_year_from_now" {
+  offset_years = 1
+}
+
+resource "konnect_system_account_access_token" "zone_system_account_token" {
+  depends_on = [konnect_system_account.zone_system_account, time_rotating.rotate, time_offset.one_year_from_now]
```

Comment:
`time_rotating` isn't added until later in the guide

Reply:
fixed in https://github.com/Kong/docs.konghq.com/pull/8553/commits/75e45b852a82a4d88fed3a0fd1a97cf4c95312de